apps/dynamodb-etl/src/main.rs

```diff
@@ -44,11 +42,11 @@ const DEFAULT_TEXT_PATH: &str = ".projectData.S";
 #[structopt(name = "dynamodb-etl", about = "", author = "")]
 struct Opt {
     /// Binary data path
-    #[structopt(short, long, default_value = ".projectBinaryData.B")]
+    #[structopt(short, long, raw(default_value = "DEFAULT_BIN_PATH"))]
```

Comment:
These are unrelated changes. I got a reply about how to do this, and tried it out. In the end, I committed here because I was tired of stashing it, and I was worried about _forgetting_ about it if I didn't unstash it.